pysyncobj/node.py

```diff
@@ -66,10 +66,26 @@ def __init__(self, address, **kwargs):
         """
 
         super(TCPNode, self).__init__(address, **kwargs)
-        self.address = address
-        self.host, port = address.rsplit(':', 1)
-        self.port = int(port)
-        self.ip = globalDnsResolver().resolve(self.host)
+        self._address = address
```

Comment:
Double underscore please (self.__address).
Single one can be easily access from outside.